main.css

```diff
@@ -142,15 +172,18 @@ h3.headline {
 }
 
 .pageHeader__secondary {
-  letter-spacing: 0.68em;
-  margin-right: -0.68em;
+  letter-spacing: 1.05em;
```

Comment:
The name here is a mistake on my part. `pageHeader__secondary` should be `pageHeader--secondary`. I might fix this and any similar mistakes in a future PR.